docs/src/fkui.css

```diff
@@ -1938,3 +1938,47 @@ code.hljs {
 .imenu--horizontal .imenu__list__anchor-span {
     padding-right: 0.75rem;
 }
+
+.ipopupmenu {
+    background-color: var(--f-background-popupmenu, #ffffff);
+}
+
+.ipopupmenu__list {
```

Comment:
Maybe we could use more descriptive class names? Or are we expecting to use more `IPopup` instances?

`.topnav__list`?

Reply:
In `fkui.css` I don't know, shouldn't it be a copy of FKUI scss?